clinica/iotools/bids_utils.py

```diff
@@ -459,11 +459,8 @@ def write_sessions_tsv(bids_dir, sessions_dict):
         if bids_id in sessions_dict:
             columns = list(list(sessions_dict.values())[0].values())[0].keys()
             session_df = pd.DataFrame(columns=columns)
-            it = 0
             for k in sessions_dict[bids_id].keys():
-                session_dict = sessions_dict[bids_id][k]
-                session_df_temp = pd.DataFrame(session_dict, index=[k, ])
-                session_df = pd.concat([session_df, session_df_temp])
+                session_df.loc[k] = sessions_dict[bids_id][k]
```

Comment:
you could even simplify this with:

```python
for k, v in sessions_dict[bids_id].items():
    session_df.loc[k] = v
```

which to me sounds like you could do it once with something like:

```python
pd.DataFrame.from_dict(sessions_dict)
```

and avoid iterating through the dictionary?

Reply:
Your simplified loop gives the expected result:
![image](https://user-images.githubusercontent.com/27341903/123240312-7e338300-d4e0-11eb-8cc2-7fd6ec2e138d.png)

Your one-line code gives an array that should be transposed: 
![image](https://user-images.githubusercontent.com/27341903/123240157-5a703d00-d4e0-11eb-8bbb-f56c78a3a181.png)

Reply:
> Your one-line code gives an array that should be transposed:

What happens if you add this option: `(..., orient="index"`)

Reply:
This modification below is OK for you?
![image](https://user-images.githubusercontent.com/27341903/123242258-20a03600-d4e2-11eb-94f6-84fa98be58ec.png)

Reply:
I think you should be able to build `session_df` in one go without the need to transpose (which is not idiomatic). Also aliasing DataFrame to DT is not great either. People may expect aliases for package names (like `pandas` -> `pd`) but usually not for datatypes.

Reply:
I had not seen your previous comment ... nicel, it works! I ll commit.

Reply:
What's the format of the `session_dict`and the expected DataFrame you'd want out of it?

Reply:
> I had not seen your previous comment ... nice, it works! I ll commit.

Great. Thanks for your patience :-)

Reply:
![image](https://user-images.githubusercontent.com/27341903/123243872-b4becd00-d4e3-11eb-970f-0416f723bfc5.png)